community/modules/scheduler/htcondor-configure/files/htcondor_configure.yml

```diff
@@ -149,6 +152,56 @@
           -token condor@{{ trust_domain }}
       args:
         creates: "{{ condor_config_root }}/tokens.d/condor@{{ trust_domain }}"
+    - name: Enable SchedD high availability
+      when: job_queue_ha | bool
+      block:
+      - name: Setup HA Spool directory
+        ansible.builtin.file:
+          path: "{{ spool_dir }}"
+          state: directory
+          owner: condor
+          group: condor
+          mode: 0700
+      - name: Set SchedD HA configuration (requires restart)
+        ansible.builtin.copy:
+          dest: "{{ condor_config_root }}/config.d/{{ schedd_ha_config_file }}"
+          mode: 0644
+          content: |
+            MASTER_HA_LIST=SCHEDD
+            HA_LOCK_URL=file:{{ spool_dir }}
+            VALID_SPOOL_FILES=$(VALID_SPOOL_FILES), SCHEDD.lock
+            HA_POLL_PERIOD=30
+        notify:
+        - Restart HTCondor
+      - name: Create SystemD override directory for HTCondor
+        ansible.builtin.file:
+          path: /etc/systemd/system/condor.service.d
+          state: directory
+          owner: root
+          group: root
+          mode: 0755
+      - name: Ensure HTCondor starts after shared filesystem is mounted
+        ansible.builtin.copy:
+          dest: /etc/systemd/system/condor.service.d/mount-spool.conf
+          mode: 0644
+          content: |
+            [Unit]
+            RequiresMountsFor={{ spool_dir }}
+        notify:
+        - Restart HTCondor
+    - name: Disable SchedD high availability
+      when: not job_queue_ha | bool
+      block:
+      - name: Remove SchedD HA configuration file
+        ansible.builtin.file:
+          path: "{{ condor_config_root }}/config.d/{{ schedd_ha_config_file }}"
```

Comment:
Should `schedd_ha_config_file` be defined at the top level `vars` block? Where is it coming from?

Reply:
Good catch. This got lost in a rebase to prepare this PR. I've fixed it and re-ran a blueprint that I've been using for manual testing. While testing that, I found that another setting ([SCHEDD_NAME](https://htcondor.readthedocs.io/en/latest/admin-manual/high-availability.html#working-with-remote-job-submission)) can be important for use cases where jobs are submitted remotely. I have added that setting.

Reply:
I have created a task to add the HA blueprint to the repo and an integration test. I believe this should wait until the 2 bugs below land in a feature release (should be 10.4 in approximately early April)

- [HTCONDOR-1590](https://opensciencegrid.atlassian.net/jira/software/c/projects/HTCONDOR/issues/HTCONDOR-1590)
- [HTCONDOR-1594](https://opensciencegrid.atlassian.net/jira/software/c/projects/HTCONDOR/issues/HTCONDOR-1594)